README.md

```diff
@@ -117,20 +123,13 @@ which leaks for the cases of syntactically undeniable Realm-specific intrinsics
 like the `AsyncFunction` constructor and prototype, and requires the
 implementer to be vigilant to the extent that they graft every intrinsic from
 one realm to another.
-We have found such arrangements to be fragile and leaky.
+We have found such arrangements to be fragile and leaky. Also costly in memory efficiency and developer time.
 
 New `Global` provide an alternate solution: evaluate modules or scripts in a
 separate global scope with shared intrinsics.
 
 ```js
-const dslGlobal = const new Global({
-  globalThis: {
-    __proto__: globalThis,
-    describe,
-    before,
-    after,
-  }
-});
+const dslGlobal = const new Global();
```

Comment:
This seemed like a leftover or a deviation from interface described earlier. 
@kriskowal